sqlglot/dialects/bigquery.py

```diff
@@ -34,12 +34,25 @@ def func(self, expression):
 
 
 class BigQuery(Dialect):
-    identifiers = ["`"]
-    escape = "\\"
     unnest_column_only = True
 
     class Tokenizer(Tokenizer):
-        QUOTES = ["'", '"', '"""']
+        QUOTES = [
```

Comment:
this can be dryed up by doing 
[
for [...]
for ['r'...]
[

Reply:
Fixed